.nycrc

```diff
@@ -1,7 +1,12 @@
 {
     "include": [
-      "packages/*/dist*/*",
-      "examples/*/dist*/*"
+      "packages/*/dist*/",
+      "examples/*/dist*/"
+    ],
+    "exclude": [
+      "packages/*/dist*/test/",
+      "examples/*/dist*/test/",
+      "**/.sandbox/"
```

Comment:
(Some) bootstrapper tests are creating the sandbox in `packages/test/.sandbox`, some tests in another location that's not covered by the two patterns above. I was not able to quickly find and fix those tests, ignoring all `.sandbox` directories seems like the best short-term option to me.